llvm/lib/Target/AMDGPU/AMDGPUTargetTransformInfo.cpp

```diff
@@ -75,6 +75,10 @@ static cl::opt<size_t> InlineMaxBB(
     cl::desc("Maximum number of BBs allowed in a function after inlining"
              " (compile time constraint)"));
 
+static cl::opt<unsigned> InlineLastCallToStaticBonus(
+    "amdgpu-inline-last-call-to-static-bonus", cl::Hidden, cl::init(165000),
```

Comment:
165000 is the default value 15000 multiplied by AMDGPU's inlining threshold multiplier, which is 11.

Reply:
What do you think about keeping it as a constant and not an option?

I don't think nobody is ever going to use that option.

And in your test the bonus contribution is already clear by comparing the scores for `@callee_not_only_one_live_use` and `@callee_only_one_live_use`.

Reply:
> What do you think about keeping it as a constant and not an option?

I don't have strong opinions on it. I'm fine with making it a constant.